web/client/codechecker_client/cmd_line_client.py

```diff
@@ -104,17 +104,16 @@ def check_run_names(client, check_names):
     if not check_names:
         return run_info
 
-    missing_name = False
     for name in check_names:
         if not run_info.get(name):
```

Comment:
Is it possible that an element of `check_names` is not in `run_info`? Just because `run_info` is based on a run filter which contains `check_names`. So is this "Possible run names are:" message ever printed?